tests/tf-manifests/rhcs/idps/openid/variables.tf

```diff
@@ -4,7 +4,7 @@ variable "token" {
 }
 variable "gateway" {
   type    = string
-  default = "https://api.stage.openshift.com"
+  default = "https://api.openshift.com"
```

Comment:
Please don't set the default gateway to production. Keep consistent with cluster's manifest. There is a global env controlling the gateway urls.

Reply:
the thing is, the `RHCS_ENV` or the `GATEWAY_URL` environment variable, does not influence the url of:
```
provider "rhcs" {
  token = var.token
  url   = var.url <-----------
}
```
it is always keep staying on "staging" because we have "url" attribute also for the idp themselves, and it overrides it in the command line vars we sent to `terraform apply`, for example:

```
resource "rhcs_identity_provider" "gitlab_idp" {
  cluster        = var.cluster_id
  mapping_method = var.mapping_method
  name           = "Gitlab"
  gitlab = {
    client_id     = var.client_id
    client_secret = var.client_secret
    url           = var.url
  }
}
```

Reply:
I will change it back, but I just hope it will not fail in CI and that this is just something that happens when running locally